website_sale_attribute_filter_category/tests/test_website_sale_attribute_filter_category.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import HttpCase
+
+
+class websiteSaleAttributeFilterCategoryHttpCase(HttpCase):
+
+    def setUp(self):
+        super(websiteSaleAttributeFilterCategoryHttpCase, self).setUp()
+        # Activate the Product Attribute's Filter for testing environment
+        with self.cursor() as cr:
+            env = self.env(cr)
+            env.ref('website_sale.products_attributes').active = True
```

Comment:
This is no longer supported thanks to https://github.com/odoo/odoo/pull/30768. You have to choose between:

1. Activate the view in the tour by clicking on "Customize". You're running the tour as admin, so it should be OK.
2. Activate the view in demo data, which is loaded before this test starts.